src/templates/replaced.tsv

```diff
@@ -1,63 +1,64 @@
-http://purl.obolibrary.org/obo/OBA_1001023	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004721	extensor digitorum longus mass
-http://purl.obolibrary.org/obo/OBA_1001024	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0005629	lung mass
-http://purl.obolibrary.org/obo/OBA_1001025	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0002707	kidney mass
-http://purl.obolibrary.org/obo/OBA_1001026	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0007027	right cardiac atrium mass
-http://purl.obolibrary.org/obo/OBA_1001027	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004722	soleus muscle mass
-http://purl.obolibrary.org/obo/OBA_1001028	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT1000641	female gonad mass
-http://purl.obolibrary.org/obo/OBA_1001029	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0002176	brain mass
-http://purl.obolibrary.org/obo/OBA_1001030	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004528	fat pad mass
-http://purl.obolibrary.org/obo/OBA_1001031	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0004257	placenta mass
-http://purl.obolibrary.org/obo/OBA_1001032	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0004951	spleen mass
-http://purl.obolibrary.org/obo/OBA_1001033	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004725	embryo mass
-http://purl.obolibrary.org/obo/OBA_1001034	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004720	gastrocnemius mass
-http://purl.obolibrary.org/obo/OBA_1001035	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0007033	heart right ventricle mass
-http://purl.obolibrary.org/obo/OBA_1001036	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT1000711	abdominal fat pad mass
-http://purl.obolibrary.org/obo/OBA_1001037	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0005027	quadriceps femoris mass
-http://purl.obolibrary.org/obo/OBA_1001038	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0005562	urinary bladder mass
-http://purl.obolibrary.org/obo/OBA_1001039	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0003402	liver mass
-http://purl.obolibrary.org/obo/OBA_1001040	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0004954	thymus mass
-http://purl.obolibrary.org/obo/OBA_1001041	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0007026	left cardiac atrium mass
-http://purl.obolibrary.org/obo/OBA_1001042	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004724	tongue mass
-http://purl.obolibrary.org/obo/OBA_1001043	owl:Class	true	http://purl.obolibrary.org/obo/OBA_0004598	fallopian tube mass
-http://purl.obolibrary.org/obo/OBA_1001044	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0007031	heart left ventricle mass
-http://purl.obolibrary.org/obo/OBA_VT0004271	owl:Class	true	http://purl.obolibrary.org/obo/OBA_VT0000572	paw/hand/foot/hoof morphology trait
+Ontology ID	Type	Obsolete	Replacement Term	label of obsolete class	Comment
+ID	TYPE	AT owl:deprecated^^xsd:boolean	AI IAO:0100001		
```

Comment:
Shouldnt the comment be compiled into OWL using `A rdfs:comment`?

Reply:
Fixed in the latest commit.